packages/core/test/foundry/optimistic-asserter/Common.sol

```diff
@@ -147,6 +158,24 @@ contract Common is Test {
             );
     }
 
+    function _wrappedAssertWithCallbackRecipientAndSs(address callbackRecipient, address escalationManager)
```

Comment:
The naming should be updated to `_wrappedAssertWithCallbackRecipientAndEm`, but plan to do it in separate PR as there are multiple references to SS across the tests.